### PR TITLE
FHAC-174: CXF Unbundling

### PR DESCRIPTION
**Pull request by christophermay07**
- "skip.unpack" config prop for Maven dependency plugin added to fix MDEP-98 error when running "dependency:analyze" goal.
- java.util.logging references removed to prevent library conflicts after CXF-unbundling.
- Pre-unbundling cleanup ONLY used to remove unused dependencies / declare used dependencies.  Library versions were not changed otherwise.

### CODE REVIEW

File: Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml [hunk 1]
```diff
@@ -86,43 +95,92 @@
...
             <groupId>javax.servlet</groupId>
             <artifactId>jstl</artifactId>
+            <scope>runtime</scope>
         </dependency>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>servlet-api</artifactId>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.servlet</artifactId>
         </dependency>
```
Comment (nareshcgi) [inline Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml:167]: Just curious, does the group id changed for this?

File: Product/Production/Adapters/AdminDistribution_a0/pom.xml [hunk 2]
```diff
@@ -20,6 +20,18 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.connectopensource</groupId>
+            <artifactId>CONNECTCommonTypesLib</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.connectopensource</groupId>
+            <artifactId>AdminDistributionWebservices</artifactId>
+        </dependency>
     </dependencies>
```
Comment (nareshcgi) [inline Product/Production/Adapters/AdminDistribution_a0/pom.xml:35]: Trying to understand how it worked without these referenced libraries before (even in some of the others poms that changed)?

File: Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml (hunk 1 above)
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml:167]: We use servlet 2.5 in some places (javax.servlet:servlet-api), and servlet 3.0 in other places (org.glassfish:javax.servlet).  In both cases, they're provided, so this is more for consistency.

File: Product/Production/Adapters/AdminDistribution_a0/pom.xml (hunk 2 above)
Comment (christophermay07) [inline Product/Production/Adapters/AdminDistribution_a0/pom.xml:35]: Originally, every module under "Adapters" inherited CONNECTCoreLib and CONNECTCommonTypesLib (amongst other dependencies) from the Adapters POM.  This is a bad practice (POM modules don't have dependencies), and was causing problems during CXF unbundling due to transitive dependencies.

Instead, each POM now declares, to the best of my ability, exactly what it needs -- no more, no less.

File: Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml [hunk 3]
```diff
@@ -86,43 +95,92 @@
         <!-- Spring framework -->
         <dependency>
             <groupId>org.springframework</groupId>
-            <artifactId>spring-core</artifactId>
```
Comment (nareshcgi) [inline Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml:89]: Any specific reason we are removing the Spring-core dependency?

Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml:89]: Our code doesn't directly use spring-core packages (org.springframework.core.*).  That jar is a transitive dependency to spring-beans, however, so the jar is still included in the lib folder in the final build.

**Comment by nareshcgi** [comment]
Reviewed the PR and looks good.

**Comment by jsmith2** [comment]
Good with PR with no comments.

**Comment by alameluchidambaram** [comment]
Reviewed PR and looks good.

**Pull request merged**